src/main/java/org/broadinstitute/consent/http/models/dataset_registration_v1/ConsentGroup.java

```diff
@@ -0,0 +1,606 @@
+    "consentGroupName",
+    "generalResearchUse",
+    "hmb",
+    "diseaseSpecificUse",
+    "poa",
+    "otherPrimary",
+    "nmds",
+    "gso",
+    "pub",
+    "col",
+    "irb",
+    "gs",
+    "mor",
+    "morDate",
+    "npu",
+    "otherSecondary",
+    "dataLocation",
+    "url"
+})
+@Generated("jsonschema2pojo")
+public class ConsentGroup {
+
+    /**
+     * Consent Group Name
+     * (Required)
+     *
+     */
+    @JsonProperty("consentGroupName")
+    @JsonPropertyDescription("Consent Group Name")
+    private String consentGroupName;
+    /**
+     * General Research Use
+     *
+     */
+    @JsonProperty("generalResearchUse")
+    @JsonPropertyDescription("General Research Use")
+    private Boolean generalResearchUse;
+    /**
+     * Health/Medical/Biomedical Research Use
+     *
+     */
+    @JsonProperty("hmb")
+    @JsonPropertyDescription("Health/Medical/Biomedical Research Use")
+    private Boolean hmb;
+    /**
+     * Disease-Specific Research Use
+     *
+     */
+    @JsonProperty("diseaseSpecificUse")
+    @JsonPropertyDescription("Disease-Specific Research Use")
+    private List<String> diseaseSpecificUse = new ArrayList<String>();
+    /**
+     * Populations, Origins, Ancestry Use
+     *
+     */
+    @JsonProperty("poa")
+    @JsonPropertyDescription("Populations, Origins, Ancestry Use")
+    private Boolean poa;
+    /**
+     * Other
+     *
+     */
+    @JsonProperty("otherPrimary")
+    @JsonPropertyDescription("Other")
+    private String otherPrimary;
+    /**
+     * No Methods Development or validation studies (NMDS)
+     *
+     */
+    @JsonProperty("nmds")
+    @JsonPropertyDescription("No Methods Development or validation studies (NMDS)")
+    private Boolean nmds;
+    /**
+     * Genetic studies only (GSO)
+     *
+     */
+    @JsonProperty("gso")
+    @JsonPropertyDescription("Genetic studies only (GSO)")
+    private Boolean gso;
+    /**
+     * Publication Required (PUB)
+     *
+     */
+    @JsonProperty("pub")
+    @JsonPropertyDescription("Publication Required (PUB)")
+    private Boolean pub;
+    /**
+     * Collaboration Required (COL)
+     *
+     */
+    @JsonProperty("col")
+    @JsonPropertyDescription("Collaboration Required (COL)")
+    private Boolean col;
+    /**
+     * Ethics Approval Required (IRB)
+     *
+     */
+    @JsonProperty("irb")
+    @JsonPropertyDescription("Ethics Approval Required (IRB)")
+    private Boolean irb;
+    /**
+     * Geographic Restriction (GS-)
+     *
+     */
+    @JsonProperty("gs")
+    @JsonPropertyDescription("Geographic Restriction (GS-)")
+    private String gs;
+    /**
+     * Publication Moratorium (MOR)
+     *
+     */
+    @JsonProperty("mor")
+    @JsonPropertyDescription("Publication Moratorium (MOR)")
+    private Boolean mor;
+    /**
+     * Publication Moratorium Date (MOR)
+     *
+     */
+    @JsonProperty("morDate")
+    @JsonPropertyDescription("Publication Moratorium Date (MOR)")
+    private String morDate;
```

Comment:
should this (and other dates) be parsed to date / validated?

Reply:
These classes are auto-generated. If we re-generate them, we need to repeat any changes we made. I had to do some already, and want to minimize that as much as possible. Additionally, the schema validation is responsible for parsing/validating the date format. For those reasons, I don't want to add more to these classes than is necessary.

Reply:
got it, makes sense. As long as it is validating the format, this is good with me.